src/error/dfxvm/self_update.rs

```diff
@@ -0,0 +1,105 @@
+use crate::error::{
+    download::{DownloadFileError, VerifyChecksumError},
+    env::NoHomeDirectoryError,
+    fs::{OpenFileError, RemoveFileError},
+    installation::InstallBinariesError,
+    json::{FetchJsonDocError, LoadJsonFileError},
+};
+use std::path::PathBuf;
+use std::process::Command;
+use thiserror::Error;
+use url::Url;
+
+#[derive(Error, Debug)]
+pub enum SelfUpdateError {
+    #[error(transparent)]
+    DownloadLatestBinaryError(#[from] DownloadLatestBinaryError),
+
+    #[error("failed to execute {command:#?}")]
```

Comment:
```suggestion
    #[error("failed to execute {command:#?}. Error: {source}")]
```
I feel like having the error available would make support on e.g. the forum much easier

Reply:
This happens automatically, because the variable is called `source`.  See https://github.com/dfinity/dfxvm/blob/main/src/log.rs#L30